Intersect.Client.Framework/Database/GameDatabase.cs

```diff
@@ -38,6 +38,8 @@ public abstract partial class GameDatabase
 
         public bool PlayerOverheadInfo;
 
+        public int CameraShakeIntensity;
```

Comment:
Considering that you have 5 options for the intensity here, i would say that an Enum list should be used instead of int values.